### ISSUE TITLE
Custom User Migrations after adding Column

### ISSUE THREAD

**Issue by doublej42** [brockallen/BrockAllen.MembershipReboot#619]
I'm using MembershipReboot with IdentityServer and I added a column 

    public class CustomUser : RelationalUserAccount
    {
        [Display(Name="First Name")]
        public virtual string FirstName { get; set; }
        [Display(Name = "Last Name")]
        public virtual string LastName { get; set; }

        public bool AllowPasswordLogin { get; set; }

    }

How do I get these changes to be reflected in the database? 

(new to code first entity framework)

**Comment by brockallen** [comment]
I think I started a sample for that here: https://github.com/brockallen/BrockAllen.MembershipReboot/blob/master/samples/CustomUserAccount/CustomUserAccount/Program.cs

As for migrations, you might want to build your own migrations. When I started MR I tried to maintain migrations in the project, but then realized that was the wrong approach. Migrations should be the app's responsibility, not the library's.
